app/src/main/java/gov/hhs/cdc/trustedintermediary/OpenApi.java

```diff
@@ -42,4 +43,13 @@ String getBaselineDocumentation() {
             throw new RuntimeException(e);
         }
     }
+
+    String getEtorDocumentation() {
+        try (InputStream openApiStream =
+                getClass().getClassLoader().getResourceAsStream("openapi_etor.yaml")) {
+            return new String(openApiStream.readAllBytes(), StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
This is giving me a code smell for the Sonar scan

Reply:
We don't need this code as the etor documentation comes in through the method parameter